### PR TITLE
Reference management models dictionary item

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to #3145

### CODE REVIEW

**Comment by busrasengul** [comment]
Thanks a lot @erikjanwestendorp 
This one also made to the list of the PRs to review tomorrow too 😊

File: Reference/Management/Models/DictionaryItem-v9.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: DictionaryItem Model
```
Comment (busrasengul) [inline Reference/Management/Models/DictionaryItem-v9.md:4]:
```suggestion
meta.Title: "DictionaryItem Model"
```

File: Reference/Management/Models/DictionaryItem-v9.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+meta.Description: Represents a Dictionary Item. A Dictionary Item is what you see in the Translation / Dictionary tree.
```
Comment (busrasengul) [inline Reference/Management/Models/DictionaryItem-v9.md:5]:
```suggestion
meta.Description: "Represents a Dictionary Item. A Dictionary Item is what you see in the Translation / Dictionary tree."
```

File: Reference/Management/Models/DictionaryItem-v9.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Constructors
+
+### new DictionaryItem(string itemKey)
+
+Constructor for creating a new `DictionaryItem` object where the necessary parameter is the key of the `DictionaryItem` as a `string`.
+
+### new DictionaryItem(Guid? parentId, string itemKey)
+
+Constructor for creating a new `DictionaryItem` object where the necessary parameters are the parentKey as `Guid` and the key of the `DictionaryItem` as a `string`. Use this one if you want to create a `DictionaryItem` untherneath another one.
```
Comment (busrasengul) [inline Reference/Management/Models/DictionaryItem-v9.md:33]:
```suggestion
Constructor for creating a new `DictionaryItem` object where the necessary parameters are the parentKey as `Guid` and the key of the `DictionaryItem` as a `string`. Use this one if you want to create a `DictionaryItem` underneath another one.
```

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: Reference/Management/Models/DictionaryItem-v9.md (hunk 3 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/DictionaryItem-v9.md]: @busrasengul Updated 😄

**Pull request merged**